src/Api/Api.tsx

```diff
@@ -0,0 +1,69 @@
+import { IErrorResponse, IMovie, IMovieById, IMovies } from '../types/interfaces';
+
+const Headers = {
+  key: 'X-API-KEY',
+};
+
+const ApiKey = '76HP0HF-JE0MXDF-G52ZG6R-M0VGAA7';
+
+const ResponseStatus = {
+  ok: 200,
+};
```

Comment:
for this small api ok, it can be there but usually this is stored in constants and API_URL in envs

Reply:
In this case I can use response.ok at all, then you won't need this variable at all.